### ISSUE TITLE
There isn't any demo, ready to run

### ISSUE THREAD

**Issue by hosseinghafarian** [rmgarnett/active_learning#1]
Dear prof Garnett,

Thank you for providing Active learning toolbox. Unfortunately, There isn't any ready to use active learning demo.

**Comment by lironsam** [comment]
+1

**Comment by rezasoll** [comment]
yes please provide us a demo to represent how to use this toolbox properly
thanks

**Comment by cplazaseco** [comment]
Has anyone finally managed to build a demo of the toolbox?
